terraform/modules/database/pgbouncer.tf

```diff
@@ -0,0 +1,97 @@
+resource "aws_ecs_task_definition" "pgbouncer" {
+  family                = "pgbouncer-service-${var.deployment_stage}"
+  requires_compatibilities = ["FARGATE"]
+  container_definitions = <<DEFINITION
+[
+  {
+    "portMappings": [
+      {
+        "hostPort": 5432,
+        "protocol": "tcp",
+        "containerPort": 5432
+      }
+    ],
+    "environment": [
+      {
+        "name": "DATABASE_URL",
+        "value": "postgresql://${var.db_username}:${var.db_password}@upload-dev.cluster-cdaogjt23uha.us-east-1.rds.amazonaws.com/upload_${var.deployment_stage}"
+      },
+      {
+        "name": "DEFAULT_POOL_SIZE",
+        "value": "20"
+      },
+      {
+        "name": "MAX_CLIENT_CONN",
+        "value": "2000"
+      },
+      {
+        "name": "POOL_MODE",
+        "value": "transaction"
+      }
+    ],
+    "memory": 1024,
+    "cpu": 512,
+    "image": "quay.io/humancellatlas/docker-pgbouncer:master",
+    "name": "pgbouncer-${var.deployment_stage}"
+  }
+]
+DEFINITION
+  network_mode          = "awsvpc"
+  cpu                   = "512"
+  memory                = "1024"
+}
+
+resource "aws_ecs_cluster" "pgbouncer" {
+  name = "pgbouncer-${var.deployment_stage}"
```

Comment:
Another naming issue.  For *any* names that are going to end up in AWS dashboards, they should be prefixed by "dcp-\<component\>" and suffixed with the deployment.  "pgbouncer-dev" is only cool if this is going to be the only pgbouncer deployed to the dev environment for all components for all time and the DCP is not sharing the AWS account with any other systems.   Try "dcp-upload-pgbouncer-dev".

Let us forestall the "whose pgbouncer is this?" conversation.